src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+@import 'variables';
+
+.card {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  overflow: hidden;
+  position: relative;
+  padding-top: 32px;
+  width: $card-width;
+  border: $card-border-color;
+  border-radius: $card-border-radius;
+  font-family: $font-family;
+  background: $hover-color;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__content {
+    padding-inline: 16px;
+    margin-top: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin: 0;
+
+    &--code {
+      font-weight: 400;
```

Comment:
it's default value - no need to set it

Reply:
fix everywhere

Reply:
> it's default value - no need to set it

If I don't set this value, it will default to 500